src/scripts/selectors.ts

```diff
@@ -58,13 +58,23 @@ const generateSelectors = async (): Promise<void> => {
   const rootSelectorCss = `@custom-selector :--root [${attr}='${value}'];`
   const rootSelectorJs = `case ":--root": return "[${attr}='${value}']"`
 
+  // Add an additional `[class*=language-]` qualified selector to Code nodes, to math PrismJS’s CSS specificity and
+  // allow styling tokenized elements.
+  const prismCompatSelector = (itemtype: string, schemaURL: string): string => {
```

Comment:
I found this choice of parameter names a little confusing as it's inconsistent with the naming in the broader context. Would prefer `type` (the name of the type) and `itemtype` (the value of the itemtype attribute on the element)